integration-test-groups/azure/azure-storage-datalake/src/main/java/org/apache/camel/quarkus/component/azure/storage/datalake/it/AzureStorageDatalakeResource.java

```diff
@@ -169,11 +178,66 @@ public String consumer(@PathParam("filesystem") String filesystem,
                 10000, String.class);
     }
 
+    @Path("/route/{route}/filesystem/{filesystem}")
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    public Object consumer(@PathParam("route") String routeName,
+            @PathParam("filesystem") String filesystem,
+            @QueryParam("useOutputStream") boolean useOutputStream,
+            Map<String, Object> headers) throws Exception {
+
+        ByteArrayOutputStream inMemoryStream = new ByteArrayOutputStream();
+
+        Map<String, Object> _headers = new HashMap();
+        if (headers != null) {
+            _headers.putAll(headers);
+
+        }
+        _headers.put("filesystemName", filesystem);
+        _headers.put("accountName", azureStorageAccountName.get());
+
+        System.out.println(_headers);
```

Comment:
Remove `println` or use a logger if the variable is worth logging.

Reply:
removed